dot/build_spec.go

```diff
@@ -94,33 +96,44 @@ func BuildFromDB(path string) (*BuildSpec, error) {
 	tmpGen.Genesis.Raw = make(map[string]map[string]string)
 	tmpGen.Genesis.Runtime = make(map[string]map[string]interface{})
 
+	// ignore the error value because there is no telemetry endpoint to connect
+	// error might happen only if there's some problem while connecting
+	disabledTelemetry, err := telemetry.BootstrapMailer(context.Background(), nil, false, nil)
+	if err != nil {
+		panic(fmt.Sprintf("telemetry should not fail at BuildFromDB function: %s\n", err.Error()))
```

Comment:
nit simpler and no need for a trailing new line with panics

```suggestion
		panic("telemetry should not fail at BuildFromDB function: " + err.Error())
```